clcache/__main__.py

```diff
@@ -469,6 +482,11 @@ def computeKeyNodirect(compilerBinary, commandLine, environment):
         compilerHash = getCompilerHash(compilerBinary)
         normalizedCmdLine = CompilerArtifactsRepository._normalizedCommandLine(commandLine)
 
+        if "CLCACHE_BASEDIR" in os.environ:
+            baseDir = normalizeBaseDir(os.environ["CLCACHE_BASEDIR"]).replace("\\", "\\\\").encode("UTF-8")
```

Comment:
Hmm, why are backslashes escaped here?

Reply:
The backslashes are escaped in the preprocessor output because `__FILE__` expands to a C string literal.